### ISSUE TITLE
reset-counters dont work

### ISSUE THREAD

**Issue by Noroby** [RubixDev/typst-i-figured#2]
```typ
#import "@preview/i-figured:0.2.0"

#show math.equation: i-figured.show-equation
#set heading(numbering: "1-1")
#show heading: i-figured.reset-counters

= heading1
$ S = pi r^2 $<eq1>
= heading2
$ S = pi r^2 $<eq2>
```
![image](https://github.com/RubixDev/typst-i-figured/assets/92096903/928e97de-02cb-4174-a4bb-fde1c8228916)

**Comment by Noroby** [comment]
I think the second number should be (2.1).

**Comment by RubixDev** [comment]
A stupid little mistake and I didn't test it before releasing :upside_down_face:. Will be fixed in 0.2.1
